### ISSUE TITLE
Checking status: 

### ISSUE THREAD

**Issue by apsankar** [ipop-project/Release-Management#6]
Once the node is up and running, we use: echo -e '\x02\x01{"m":"get_state"}' | netcat -q 1 -u 127.0.0.1 5800 to see all connected nodes. However, the 'netcat' command used here belongs to OpenBSD. centOS uses 'ncat', which is supposed to be newer; they aren't compatible. So the -q flag which signifies to read from stdin till EOF is reached doesn't seem to have an equivalent in 'ncat'. Need to work around this.

**Comment by vdaneshmand** [comment]
@kcratie ,
If you don't see any problem with this, I'm going to use `tcpdump` command instead of `netcat` or `ncat` to get the status of the nodes. It works on both Ubuntu and CentOS and probably all other major distributions of Linux and this can solve the problem.

**Comment by kcratie** [comment]
The important thing is to retrieve the status of the node. Once tcpdump does this, it is an acceptable short term solution.

**Comment by vdaneshmand** [comment]
@kcratie 
I replaced `netcat` command with `tcpdump` and got the same results. As a sample, this is the content of the package both `netcat` and `tcpdump` capture and output as the result of `./ipop status` command:

```
`......@...................................y...-..{
   "_fpr" : "58:F3:F3:C2:59:AA:02:E3:F5:4C:A8:88:12:4F:E3:2E:CF:55:03:7A",
   "_ip4" : "1.1.1.1",
   "_ip6" : "fd50:0dbc:41f2:4a3c:4096:29a0:8b9b:3f3b",
   "_mac" : "12:b6:9b:14:6d:ce",
   "_uid" : "409629a08b9b3f3be610b8832cc28822f964410f",
   "type" : "local_state"
}
```
I think it is much better to reformat it and make it more readable.

**Comment by kcratie** [comment]
@birdyhuang Please verify that this works on ContOS. Thanks!

**Comment by vdaneshmand** [comment]
I verify it works on both CentOS and Ubuntu. Check status is not available in the v16.08, though.
